src/Kodify/BlogBundle/Resources/views/Post/view.html.twig

```diff
@@ -9,5 +9,34 @@
         <div class="panel-body">
             {{ post.content }}
         </div>
+        <div class="panel-footer">
+            {% include 'KodifyBlogBundle:Post:Component/rating.html.twig' %}
+        </div>
     </div>
-{% endblock %}
\ No newline at end of file
+
+    <section id="comments">
+        <header>
+            <h1>Comments</h1>
+        </header>
+        <ul class="list-unstyled">
+            {% for comment in post.comments %}
+                <li class="comment">
+                    <div>
+                        <span class="comment-author">{{ comment.author.name }}</span>
+                    </div>
+                    <div>
+                        <span class="comment-text">{{ comment.text }}</span>
+                    </div>
+                    <div class="comment-date">
+                        {{ comment.createdAt|date('d/m/Y H:i') }}
+                    </div>
+                </li>
+            {% else %}
+                <li class="no-comments">There are no comments</li>
+            {% endfor %}
+        </ul>
+        <footer>
+            <a href="{{ path('comment_post', {'id': post.id}) }}" class="new-comment btn btn-success">Comment</a>
```

Comment:
The comment box should be there, I begin to care about time and didn't do it. Also, the sentence in the scenario "I press comment" and after, "I press publish" (or something like this) made my think that you want a two steps feature. But of course, you shouldn't leave the page to comment a post... This should be an AJAX feature. Also, we can do it in the same page without AJAX but I think AJAX is perfect for this use case.